docs/advanced.rst

```diff
@@ -758,6 +758,35 @@ Please refer to the supplemental example for details.
     length queries (``__len__``), iterators (``__iter__``), the slicing
     protocol and other kinds of useful operations.
 
+C++11 chrono datatypes
+======================
+When including the additional header file :file:`pybind11/chrono.h` conversions from c++11 chrono datatypes
+to corresponding python datetime objects are automatically enabled.
+The following rules describe how the conversions are applied.
+
+When passed to python objects of type ``std::chrono::system_clock::time_point`` are converted into datetime.datetime objects.
```

Comment:
Grammar fix for this and the next two paragraphs:

When C++ objects of type [..] are passed [..] they are [..].

``datetime.datetime`` needs backticks for formatting.